cluster/centos/config-build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Contains configuration values for the Binaries downloading and unpacking.
+
+# Directory to store release packages that will be downloaded.
+RELEASES_DIR=${RELEASES_DIR:-/tmp/downloads}
+
+# Define flannel version to use.
+FLANNEL_VERSION=${FLANNEL_VERSION:-"0.4.0"}
```

Comment:
can we use 0.5.3 or latest? 0.5.3 works for me, and we did fix some bugs affecting to our CentOS 7 deployment since 0.4.0 :-)
see: https://github.com/coreos/flannel/releases for details